src/api.ts

```diff
@@ -35,8 +36,28 @@ export const graphql = (body: {
     });
 };
 
-export const requestTx = (tx: any): Promise<any> =>
-  graphql({
+export const requestTx = (transaction: any, ots: RequestSignTxOptions): Promise<any> => {
+  const { broadcast, payer,  addAuths} = ots
+
+  const input: any = {
+    transaction,
+    trxOpt: {
+      broadcast,
+    }
+  };
+
+  if(!broadcast && payer && addAuths) {
+    // add additional authorization
+    input.addAuths = addAuths;
+
+    // add payer
+    input.transaction.actions[0].authorization.push({
```

Comment:
@duyphanz can we just push `addAuths` here and remove payer since `addAuths` seems to be more generous?